_includes/footer.html

```diff
@@ -1,12 +1,26 @@
 <footer>
     <div class="container">
         <div>
+            {% if site.newsletter %}
             <a class="no-ext-link-icon" href="{{ site.newsletter }}">
-            <span class="fas fa-fw fa-envelope-open-text"></span> {{ site.data.lang.footer.get-newsletter }}</a><br/>
+                <span class="fas fa-fw fa-envelope-open-text"></span> {{ site.data.lang.footer.get-newsletter }}</a><br/>
```

Comment:
nit: why is this indented and the other instances not?

Reply:
That was just a typo, fixed.